src/Query/Expressions/SelectExpression.cs

```diff
@@ -131,12 +134,15 @@ public void SetTableExpression(TableExpression tableExpression)
         public void SetWhereExpression(WhereExpression whereExpression)
         {
             Check.NotNull(whereExpression, nameof(whereExpression));
-            if (WhereExpression != null)
+
+            if (WhereExpression is null)
```

Comment:
Documentation is not updated. Method does not throw anymore and it should be noted, that multiple calls append the condition using `AND`.